experimenter/experimenter/nimbus_ui/tests/test_views.py

```diff
@@ -2988,6 +2988,102 @@ def test_render_to_response(self):
         self.assertEqual(response.context["experiment"], experiment)
         self.assertTemplateUsed(response, "nimbus_experiments/results.html")
 
+    @parameterized.expand(
+        [
+            (
+                {
+                    "v3": {
+                        "other_metrics": {"group": {"metricA": "Metric A"}},
+                        "metadata": {
+                            "metrics": {"metricA": {"friendlyName": "Friendly Metric A"}},
+                        },
+                        "overall": {
+                            "enrollments": {"all": {}},
+                            "exposures": {"all": {}},
+                        },
+                        "weekly": {
+                            "enrollments": {"all": {}},
+                            "exposures": {"all": {}},
+                        },
+                    }
+                },
```

Comment:
If you wanted to be more concise, you could keep the static parts in the function and only put the dynamic stuff (the parts that go in `weekly` and `overall`) in here. This is OK for now though.